pkg/db-repo/notification_repo.go

```diff
@@ -77,10 +77,7 @@ func (r *notifyingRepository) doCallback(ctx context.Context, callbackType strin
 	}
 
 	if len(errors) > 0 {
-		err := fmt.Errorf("there were %v errors during execution of the callbacks for create", len(errors))
-		logger.Error(err, err.Error())
-
-		return err
+		return fmt.Errorf("there were %v errors during execution of the callbacks for create", len(errors))
```

Comment:
Maybe we should replace the whole error by a multierror? I for my part hate it if I know there were 7 errors, but don't know what the errors were...

Reply:
using multierror now